docs: explain pg_stat_statements in YSQL extensions.md (#5079)
yugabyte/yugabyte-db: docs/content/latest/api/ysql/extensions.md

For more information see [`pgcrypto`](https://www.postgresql.org/docs/current/pgcrypto.html) in the PostgreSQL Docs.

### pg_stat_statements

> The `pg_stat_statements` is installed by default and needs to be enabled before it can be queried. 

Reviewer: ```suggestion
The [`pg_stat_statements`](https://www.postgresql.org/docs/11/pgstatstatements.html) extension module is installed by default, but must be enabled before the `pg_stat_statements` view can be queried. 
```

```postgresql
CREATE EXTENSION pg_stat_statements;
SELECT query, calls, total_time, min_time, max_time, mean_time, stddev_time, rows FROM pg_stat_statements;
```

> The same info is also available in the UI. On every `yb-tserver` node, you can access `http://<yb-tserver-ip>13000

Reviewer: ```suggestion
The same information is also available in the YB-TServer Admin UI. On any `yb-tserver` node, go to `http://<yb-tserver-ip>:13000` to access the data.
```

> The same info is also available in the UI. On every `yb-tserver` node, you can access `http://<yb-tserver-ip>13000
> /statements` for output of pg_stat_statements
> in `json` format. 

Reviewer: ```suggestion
You can also access the output of `pg_stat_statements` (in JSON format) by using a web browser to go to `http://<yb-tserver-ip>:13000/statements`, where yb-tserver-ip* is the address of any YB-TServer node of your cluster.
```

> For more information see [`pg_stat_statements`](https://www.postgresql.org/docs/11/pgstatstatements.html) in the PostgreSQL Docs.

Reviewer: ```suggestion
For more information, see [`pg_stat_statements`](https://www.postgresql.org/docs/11/pgstatstatements.html) in the PostgreSQL documentation.
```

> ### spi module

Reviewer: ```suggestion
### Server Programming Interface (spi) module
```

> The `spi` module includes several separate extensions using the Server Programming Interface (SPI) and triggers.

Reviewer: ```suggestion
The `spi` module lets developers use the [Server Programming Interface (SPI)]() to use the C programming language to create user-defined functions and stored procedures and to run YSQL queries directly against YugabyteDB. In YugabyteDB, the following four (of the five) extensions provided in the `spi` module are supported:

includes four groups of functions provided as separately-installable extensions that use the Server Programming Interface (SPI) and triggers.
```

> The specific extensions currently supported in YSQL are:

Reviewer: Remove this line

> - `insert_username`: functions for tracking who changed a table

Reviewer: ```suggestion
- `insert_username`: Functions for tracking who changed a table.
```

Reviewer: Capitalize the first letter and add a period to the other three extensions. [For some reason, the GitHub suggested change feature is not letting me select them.]
